src/index.js

```diff
@@ -8,18 +8,22 @@ import { BrowserRouter } from "react-router-dom";
 import "./index.css";
 import configureStore from "./state/store/configureStore";
 import { Provider } from "react-redux";
+import "./i18n"
 
 axios.defaults.baseURL = process.env.REACT_APP_API_URL;
 
 const store = configureStore();
 window.store = store;
 
 ReactDOM.render(
-  <Provider store={store}>
+  <Suspense fallback={<div>Loading.... Mauw</div>}>
```

Comment:
the `Suspense` should only wrap the `App`

Reply:
we have moved it inside Provider, but still outside of BrowserRouter